charts/cortx/templates/_cluster.yaml.tpl

```
@@ -1,7 +1,13 @@
+### TODO Revisit UUID defaults here since we are moving away from UUID entirely...
 {{- define "storageset.node" -}}
 - name: {{ .name }}
+  {{- if eq .type "server_node" }}
+  id: {{ default uuidv4 .id | quote }}
```

```suggestion
  id: {{ required "A valid id is required for server nodes" .id | quote }}
```
You're already guarding this from being called when the UUID is not defined, so `default` is never going to occur. Using `required` would make it obvious.